### ISSUE TITLE
Add a result report after completing ping

### ISSUE THREAD

**Issue by kx001001** [austoonz/FastPing#2]
How about to add a result report after completing ping? For example, Average/Max/Min RTT, total sent packets, total received packets, packet loss packets/rate...

**Comment by austoonz** [comment]
Thanks for the suggestion, and I agree. These would likely be far more useful than a simple `RoundtripAverage`.

Here's a sample of what I'm thinking...

```text
# Updated call specifying Count and RoundtripAveragePingCount
PS>$results = Invoke-FastPing -HostName '1.1.1.1','amazon.com' -Count 1 -RoundtripAveragePingCount 50

# The default output
PS>$results

HostName         Online Status     p90   PercentLost
--------         ------ ------     ---   -----------
1.1.1.1          True   Success    6     2
amazon.com       True   Success    72    0

# The complete output
PS>$results | Format-Table *

HostName          Online  Status Sent Received Lost PercentLost Min p50 p90 Max RawValues
--------          ------  ------ ---- -------- ---- ----------- --- --- --- --- ---------
1.1.1.1             True Success   50       49    1           2   4   5   6   7 {5, 4, 5, 4…}
amazon.com          True Success   50       50    0           0  70  72  72  72 {72, 72, 72, 72…}

# `RoundtripAverage` still exists as a hidden property to maintain backwards compatiblity in the output object.
# It is hidden by default as the other values are likely more useful.
PS>$results | Select-Object HostName, Status, RoundtripAverage

HostName           Status RoundtripAverage
--------           ------ ----------------
1.1.1.1           Success             5.04
amazon.com        Success            71.66
```

What do you think?
Anything additional you'd like to see?
Thoughts on removing `RoundtripAverage` from the default output in favor of the other values?

**Comment by kx001001** [comment]
Great!! Looks good…no more comments.

________________________________

**Comment by austoonz** [comment]
Awesome! I'll finalize that in the next day or so and will get it published to the gallery.

**Comment by austoonz** [comment]
I've just published FastPing v0.6.0 to the [gallery](https://www.powershellgallery.com/packages/FastPing/0.6.0) with this change. 

Thanks for the suggestion!
